### ISSUE TITLE
Feature Request: Document building this repository

### ISSUE THREAD

**Issue by joeatwork** [flatpak/flatpak#141]
Thank you for making this repository available to the community!

It would be useful to have a short document describing how to build these sources. That is, something like

``` sh
git clone https://github.com/flatpak/flatpak.git
cd flatpak
# do something else...
make
```

**Comment by hadess** [comment]
```
./autogen.sh --prefix etc. etc.
```

The same as nearly every Free Software package under the sun. You should probably be using your distributions' packages if you stumbled at this point, it's system software that requires integration, not really something you want to be building yourself.

**Comment by joeatwork** [comment]
Thanks for your response! "we use buildconf, and building requires GTK version X.Y.Z, along with the following library and tooling version dependencies" might be enough to get contributors going.

**Comment by mwleeds** [comment]
This was addressed in commit 9bb4bc8df. @joeatwork for installing dependencies you could use `apt-get build-dep flatpak` or `dnf builddep flatpak`.
